src/activities/discussions/DiscussionTopicEntity.js

```diff
@@ -64,9 +157,11 @@ export class DiscussionTopicEntity extends Entity {
 		if (!topic) return;
 
 		const updateNameAction = this._formatUpdateNameAction(topic);
+		const updateDescriptionAction = this.canEditDescription() ? this._formatUpdateDescriptionAction(topic) : null;
```

Comment:
Can you move the permission check into the format action method to keep it consistent with the above line? I think it would help with readability 😄